.github/workflows/ci_integration.yml

```diff
@@ -45,6 +45,7 @@ jobs:
           config_file: ".markdownlint.yml"
           files: "nautobot examples *.md"
           ignore_files: "nautobot/project-static"
+          rules: "scripts/use-relative-md-links.js"
```

Comment:
Naive question: this is just specifying *additional* rules right? All of the baseline markdownlint rules will still be enforced?

Reply:
Correct, https://github.com/nosborn/github-action-markdown-cli#inputs

```
files - what to process (files, directories, globs)
config_file (optional) - configuration file (JSON or YAML)
ignore_files (optional) - files to ignore/exclude (file, directory, glob)
ignore_path (optional) - path to file with ignore pattern(s)
rules (optional) - custom rule files (file, directory, glob, package)
```